sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -436,6 +436,37 @@ private int countLogs(String whereClause, String ...whereArgs) {
         return count;
     }
 
+    @Override
+    public long getOldestLogTime(@NonNull String group) {
+        Long lastTimestamp = 0L;
+
+        /* Log. */
+        AppCenterLog.debug(LOG_TAG, "Trying to get the time of the oldest log from the Persistence database for " + group);
+
+        /* Query database. */
+        SQLiteQueryBuilder builder = SQLiteUtils.newSQLiteQueryBuilder();
+        builder.appendWhere(COLUMN_GROUP + " = ?");
+        Cursor cursor = null;
+        String[] selectionArgsArray = {group};
+        String[] columnArgsArray = {String.format("min(%s)", COLUMN_TIMESTAMP)};
```

Comment:
min? At the first look, should be max...

Reply:
yes, because we must get the oldest log but no last